### DISCUSSION TITLE
Search Filter: Organizations

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24004] [New to GitHub]
<p>Although this thought may be similar to <a href="https://github.community/t/feature-request-easier-organization-searching/140023">another post</a>, it may be more beneficial if, instead, we had a filter for organizations on the result page. For example, looking up Microsoft will give you everything related to it - except the tab that lists the group(s). It could save developers a lot of time from having to dig through the <code>Users</code> section.</p>
<p><img src="https://user-images.githubusercontent.com/10137/181080592-07f129cd-cd55-4e5e-9f15-6eb4303a00bb.png" alt="gh_microsoft" data-base62-sha1="e5yoOI5VKRLDpEBznMTgkGthvFW" width="177" height="288"></p>

**Comment by strafe** [comment, marked as answer]
<p>Checkout my post on the linked topic, it may accomplish what you need: <a href="https://github.community/t/feature-request-easier-organization-searching/140023/4" class="inline-onebox">Feature Request: Easier Organization Searching - #4 by strafe</a></p>

**Comment by ghost** [comment]
<p>Good to know - I’ll mark this as the solution, but I do wish this tip/filter was more accessible.</p>
